pkgs/os-specific/linux/drbd/driver.nix

```
@@ -1,12 +1,12 @@
-{ stdenv, lib, fetchurl, kernel, flex, coccinelle, python3 }:
+{ stdenv, lib, fetchurl, kernel, nixosTests, flex, coccinelle, python3 }:
 
-stdenv.mkDerivation rec {
-  name = "drbd-${version}-${kernel.version}";
-  version = "9.2.8";
+stdenv.mkDerivation (finalAttrs: {
+  pname = "drbd";
+  version = "9.2.9";
 
   src = fetchurl {
-    url = "https://pkg.linbit.com//downloads/drbd/9/drbd-${version}.tar.gz";
-    hash = "sha256-LqK1lPucab7wKvcB4VKGdvBIq+K9XtuO2m0DP5XtK3M=";
+    url = "https://pkg.linbit.com//downloads/drbd/9/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
```

```suggestion
    url = "https://pkg.linbit.com//downloads/drbd/9/drbd-${finalAttrs.version}.tar.gz";
```
finalAttrs.pname is in many usecases wrong, as when we would append something like headless to pname, then the download URL would be broken

---

Thanks for the hint. What would be the correct variable and do you maybe have a link to the related documentation?

---

Don't use pname, just use the string literal.